cocos/2d/CCCamera.cpp

```diff
@@ -539,4 +539,10 @@ int Camera::getRenderOrder() const
     return result;
 }
 
+void Camera::visit(Renderer* renderer, const Mat4 &parentTransform, uint32_t parentFlags)
```

Comment:
There may be some bug if the camera is visited after other objects (such as sprite etc). In this case the update of `_viewProjectionUpdated ` is too late.